tests/tcp.rs

```diff
@@ -175,9 +175,11 @@ fn accept_front_of_line_blocking() -> Result {
     sim.host("B", || async {
         let listener = bind().await;
 
-        while let Ok((_, peer)) = listener.accept().await {
-            tracing::debug!("peer {}", peer);
-        }
+        tokio::spawn(async move {
```

Comment:
this is probably a reasonable test, but i think you can also copy tokio's assert send/sync macro into turmoil and make these tests first-class: https://github.com/tokio-rs/tokio/blob/master/tokio/tests/async_send_sync.rs#L117

Reply:
TIL, thanks for sharing!